requirements-app.txt

```diff
@@ -15,7 +15,7 @@ celery[sqs]>=4.4.7
 click-datetime>=0.2
 
 # Feb 2022: Package for Datadog Tracer
-ddtrace==1.8.0
+# ddtrace==1.8.0
```

Comment:
```suggestion
ddtrace==1.8.0
```

Reply:
sorry, did this in baldwin's branch and not here